app/views/schools/choose_programme/edit.html.erb

```diff
@@ -0,0 +1,48 @@
+<% content_for :before_content, govuk_back_link( text: "Back", href: schools_cohort_path(@cohort.start_year)) %>
```

Comment:
I think this needs a page title

Reply:
@amazimbe Might be. Do we have any consistent title naming convention, @ethanmills Thoughts?

Reply:
Title is " Your induction programme". These can be found in the content doc